site/docs/spec.md

```diff
@@ -254,6 +255,33 @@ Notes:
 
 1. Technically, data files can be deleted when the last snapshot that contains the file as “live” data is garbage collected. But this is harder to detect and requires finding the diff of multiple snapshots. It is easier to track what files are deleted in a snapshot and delete them when that snapshot expires.
 
+#### Deletion Files
```

Comment:
Why is this a sub-section of _Manifests_?

Let's create a new section after _Table Metadata_ called _Delete Format_, with a sub-section for _Position-based delete files_. Then we should note that content in that section is in preparation for incrementing the format version:

> This section details how to encode row-level deletes in Iceberg metadata. Row-level deletes are not supported in the current format version, 1. This part of the spec is not yet complete and will be completed as format version 2.

Reply:
no problem.